docs/reference/HailExpressionLanguage.md

```diff
@@ -109,19 +109,22 @@ Several Hail commands provide the ability to perform a broad array of computatio
      - exists: `set.exists(v => expr)` -- Returns a boolean which is true if **any** element satisfies `expr`, false otherwise
      - forall: `set.forall(v => expr)` -- returns a boolean which is true if the set is empty, or `expr` evaluates to `true` for **every** element
 
- - Dict Operations:
-     - select: `dict[key]` -- returns the value keyed by the string `key`.  An example might be `global.genedict["SCN2A"]`.
-     - contains: `dict.contains(key)` -- returns true if `dict` has key `key`, false otherwise.
-     - mapValues: `dict.mapValues(x => expr)` -- returns a new dict with a transformation of the values
-     - size: `dict.size` -- returns the number of key/value pairs
-     - isEmpty: `dict.isEmpty` -- returns true if there is at least one key/value pairs
+ - Map Operations:
+     - select: `map[key]` -- returns the value keyed by the string `key`.  Example: `global.genemap["SCN2A"]`.  This method will fail if the key is not contained in the map.
+     - get: `map.get(key)` -- identical to `select`, but returns missing if the key is not found, rather than failing.
+     - contains: `map.contains(key)` -- returns true if `map` has key `key`, false otherwise.
+     - mapValues: `map.mapValues(x => expr)` -- returns a new map with a transformation of the values
+     - size: `map.size` -- returns the number of key/value pairs
+     - isEmpty: `map.isEmpty` -- returns true if there is at least one key/value pairs
+     - keys: `map.keys` -- returns an `Array` of the map keys
+     - keySet: `map.keySet` -- returns a `Set` of the map keys
```

Comment:
Actually, both scala and python return a non-unique collection type (Array + list).  We should do that too.